pino.js

```diff
@@ -48,6 +50,14 @@ const defaultOptions = {
   serializers: Object.assign(Object.create(null), {
     err: defaultErrorSerializer
   }),
+  formatters: Object.assign(Object.create(null), {
```

Comment:
I don't know when we started creating null objects for default options but it's completely unnecessary

Reply:
They are way faster.

Reply:
wtf. I thought they were slower

and also .. wtf.